docs/python-grader/index.md

```diff
@@ -150,11 +150,26 @@ The server parameters in `data` can be accessed from within the test cases using
 
 If the optional files `tests/leading_code.py` and/or `tests/trailing_code.py` exist, the autograder will automatically prepend and append the contents to the user's submission before grading. This can be useful if alternative input methods are used, e.g. for Parson's problem questions to provide python imports or other code that must be run.
 
-#### Multiple Iterations
+### Using `__name__ == "__main__"`
+
+By default, the student's code does not execute with `__name__` set to `"__main__"`, as is usually the case with Python scripts that are executed as a Python script. This allows students to use code like the following to test and debug their code in their local environment without affecting the autograder functionality:
+
+```python
+if __name__ == "__main__":
+    # Student's own debugging code
+```
+
+If, on the other hand, a question expects a student to write code that checks for the value of `__name__`, this can be achieved by adding a [tests/leading_code.py file](#leading-and-trailing-code) with the following content:
```

Comment:
```suggestion
If, on the other hand, a question expects a student to write code that checks for the value of `__name__`, this can be achieved by adding a [`tests/leading_code.py` file](#leading-and-trailing-code) with the following content:
```